pkg/controllers/live.go

```diff
@@ -2,18 +2,27 @@ package controllers
 
 import (
 	"encoding/json"
+	"log"
+	"net/http"
+
 	"github.com/gorilla/mux"
 	"github.com/smartatransit/third_rail/pkg/clients"
 	"github.com/smartatransit/third_rail/pkg/transformers"
 	"github.com/smartatransit/third_rail/pkg/validators"
-	"log"
-	"net/http"
 )
 
 type LiveController struct {
 	MartaClient clients.MartaClient
 }
 
+// GetScheduleByLine godoc
```

Comment:
These are the annotations that describe the documentation -- I may have missed some parameters or gave a poor description